inst/input/hector_ssp126.ini

```diff
@@ -51,7 +51,7 @@ soil_c=917   		; initial soil pool, Pg C
 ;nonpf.soil_c=609
 
 ; Permafrost is active by default in Hector v3
-; See Woodard et al. 2021, 10.5194/gmd-14-4751-2021
+; See Woodard et al. 2021, 10.5194/gmd-14-4751-2021orh
```

Comment:
This seems wrong...the old DOI resolves, but this new one does not.

Reply:
That is 100% wrong most likely a fat-fingered typo on my part! Thanks for catching this